src/Symfony/Component/Process/Process.php

```diff
@@ -303,11 +304,11 @@ public function start(callable $callback = null, array $env = [])
             $commandline = $this->replacePlaceholders($commandline, $env);
         }
 
-        $options = ['suppress_errors' => true];
-
         if ('\\' === \DIRECTORY_SEPARATOR) {
-            $options['bypass_shell'] = true;
-            $commandline = $this->prepareWindowsCommandLine($commandline, $env);
+            if (!$this->options['create_new_console']) {
```

Comment:
I added this "if" to avoid the execution of `prepareWindowsCommandLine` which will prevent the main process to exit if the child process created with `create_new_console`  has not terminated in which case I don't see a good reason to use `create_new_console` at all.

Reply:
As far as I tested correctly, calling prepareWindowsCommandLine doesn't block the main process. I ran a "sleep 2" in a new console and the main process stopped before the sleep.
Do you have a way to reproduce the behavior you describe?

Reply:
I checked again now, you're right, `prepareWindowsCommandLine` doesn't seem to stop the main process from exiting. So I can remove that check.